sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 - Removes the `@azure/ms-rest-nodeauth` dependency.
   This allows users to use any version of `@azure/ms-rest-nodeauth` directly with `@azure/service-bus` without TypeScript compilation errors.
   Fixes [bug 8041](https://github.com/Azure/azure-sdk-for-js/issues/8041).
+- Relaxes the scheme check for the endpoint while parsing the connection string.
+  This allows "\<anything\>://" as the scheme as opposed to the rigid "sb://" scheme suggested by the connection string in the portal.
```

Comment:
```suggestion
  This allows "\<anything\>://" as the scheme as opposed to the "sb://" scheme suggested by the connection string in the portal.
```

For some reason having 'rigid' in there makes me think that they're suggesting something "incorrect" which isn't really true either.